ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1038,6 +1038,15 @@ FCU_SS_ZTM:
   - SS
   - ZTM
 
+FCU_SS_SDM:
```

Comment:
Does this FCU have a fan? Just trying to see if this equipment could fall under an existing FCU type but most existing types include fan points.

Reply:
@ccquigley - Unfortunately no FANs on it. Just the SS and supply damper monitoring points to exchange the air. Thank you, Ciaran

Reply:
Can we get more information on what the start stop is for then? Is there a fan but we don't have points or is this for heating/cooling? Also is this a landlord unit?